### DISCUSSION TITLE
Nginx Reverse Proxy

### DISCUSSION THREAD

**Discussion by davbay1** [chuot/rdio-scanner#185] [Q&A]
Hi everyone,

I accidently posted this in the wrong place so I am reposting here.

Has anyone been able to get Rdio-Scanner to work through a Nginx reverse proxy? When I am on lan and go to the web ui via the ip, it works great. However, when I try to access using the domain name through the reverse proxy, I see the web ui but it says no link and I am unable to go any further.

JayManCanada6014 commented that I need to have WebSockets Support turned on and I do. Do I need to add something else to my proxy config?

**Comment by GitGoorilla** [comment]
Mine functions correctly. I run mine through a domain and CloudFlare (to hide my public IP, etc). 

How do you have it setup in Nginx? Are your ports forwarded on your router (443 for SSL)?

Here are my current settings in Nginx. 

Edit: Photos/Text
![Screenshot 2022-05-21 180722](https://user-images.githubusercontent.com/88252409/169671710-2a2905fe-7e2c-4325-86ee-73c79a5b9742.png)
 
![image](https://user-images.githubusercontent.com/88252409/169671757-7489c09b-617e-450b-9999-1a979196149b.png)

**Reply by davbay1** [reply]
I have the exact same setup. Running through a domain and CloudFlare.

All ports are forwarded. Everything else I have forward so far works great.

I am also using NPM and have all the same setting checked including Websocket Support.

The only difference might be the fact that I am running NPM in a docker in Unraid and Rdio in a VM.
